migrations/server/mysql/0002_role_on_keys.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `timestamp_keys` ADD COLUMN `role` VARCHAR(255) NOT NULL, DROP KEY `gun`, ADD UNIQUE KEY `gun_role` (`gun`, `role`);
```

Comment:
I don't know if this will work, but we probably also want to add a DEFAULT 'timestamp' to the alter table, else all our existing rows will have an empty role.  We could, in the code, assume that an empty role means "timestamp role" but it'd be awesome to do this in the migration, if it works.